From 4b3c5ece623af2403fe1b82c79d44a951501ac70 Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Fri, 31 Aug 2018 09:55:08 +0200
Subject: [PATCH] Disable gold-linker for host builds

To avoid error:
| FAILED: host/brotli
...
| collect2: fatal error: cannot find 'ld'
| compilation terminated.

Signed-off-by: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
---
 src/buildtools/configure_host.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/buildtools/configure_host.pro b/src/buildtools/configure_host.pro
index 70161c82..ac59ae65 100644
--- a/src/buildtools/configure_host.pro
+++ b/src/buildtools/configure_host.pro
@@ -12,7 +12,7 @@ GN_OS = $$gnOS()
 clang: GN_CLANG = true
 else: GN_CLANG = false
 
-use_gold_linker: GN_USE_GOLD=true
+use_gold_linker: GN_USE_GOLD=false
 else: GN_USE_GOLD=false
 
 GN_V8_HOST_CPU = $$GN_HOST_CPU
-- 
2.16.4

